tests/test_frictionless_to_ckan.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+
+import pytest
+
+import frictionless_ckan_mapper.frictionless_to_ckan as frictionless_to_ckan
+
+
+converter = frictionless_to_ckan.FrictionlessToCKAN()
+
+
+class TestResourceConversion:
+    pass
+
+
+class TestPackageConversion:
+    def test_name_title_and_version(self):
+        indict = {
+            'name': 'gdp',
+            'title': 'Countries GDP',
+            'version': '1.0',
+        }
+        result = converter.package(indict)
+        assert result['name'] == indict['name']
+        assert result['title'] == indict['title']
+        assert result['version'] == indict['version']
+
+    def test_name_is_lowercased(self):
```

Comment:
let's not make any changes to name etc. Let's just do a clean map/passthrough.

Reply:
I assume this is a copy over - but noting for the future.

Reply:
> I assume this is a copy over - but noting for the future.

Yes: I started with the existing tests to copy them over. Those were just simplified a bit to use the correct syntax with pytest (different `assert` statement) and remove the dependency for the package `datapackage` so we don't need to import it anymore (it's now using a simple Python dictionary instead of creating a data package which comes with more attributes and metadata that we don't need for those tests).